e2e/cypress/fixtures/message-attachments-with-menu-options.json

```diff
@@ -0,0 +1,72 @@
+{
+    "simple-list": [{
```

Comment:
`simple-list` is not used elsewhere.  Do you have plan to replace the existing ones?

Reply:
I have removed this for now,
After all the PRs of mine are merged i plan to change simpleoptions from this file instead